Sources/FoundationInternationalization/Calendar/Calendar_ICU.swift

```diff
@@ -1159,7 +1159,11 @@ internal final class _Calendar: Equatable, @unchecked Sendable {
 
             for i in 0..<7 {
                 var status = U_ZERO_ERROR
-                weekdayTypes[i] = ucal_getDayOfWeekType(ucalendar, UCalendarDaysOfWeek(rawValue: weekdaysIndex[i]), &status)
+#if os(Windows)
```

Comment:
Does it make sense instead here to provide an extension on `UCalendarDaysOfWeek` in our ICU shims file which takes either type?

Reply:
(same for the other ifdefs)

Reply:
You cannot provide an overload in C IIRC.  We could change the signature to take a signed integer, and always cast to the signed type.  I'm not sure if that would be difficult to maintain or not.

Reply:
In general I'm wondering if we can isolate the platform difference to something closer to the ICU "API" rather than having to put it in places that use it, which theoretically are abstracted from that. It may be that we need a slightly thicker layer of abstraction in the first place.

Reply:
How about something like an underscored variant of the API that is shared?  I'll try to sketch out something.

Reply:
The enums are more difficult than I anticipated ... maybe you have an idea?